css/styles.css

```diff
@@ -510,3 +529,85 @@ header {
   padding-left: 0;
   margin: 4px 0;
 }
+
+footer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+footer .contact-form {
+  width: calc(100% - 24px);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin: 100px 24px;
+}
+
+.contact-form .form-description {
+  font-family: 'Roboto', sans-serif;
```

Comment:
- [ ] To be more in line with the Figma design, kindly use the same font for your form and form elements, in the case of your template, it seems to be Inter

![image](https://user-images.githubusercontent.com/73140461/144148079-7675c63f-e983-4be1-a294-694eaf875bc2.png)

Reply:
I am going to change it.